Makefile

```diff
@@ -164,81 +164,127 @@ help:
 local-cloud-build: gcloud
 	cloud-build-local --config=cloudbuild.yaml --dryrun=false $(LOCAL_CLOUD_BUILD_PUSH) --substitutions SHORT_SHA=$(VERSION_SUFFIX),_GCB_POST_SUBMIT=$(_GCB_POST_SUBMIT),BRANCH_NAME=$(BRANCH_NAME) .
 
-push-images: push-service-images push-client-images push-mmf-example-images push-evaluator-example-images
-push-service-images: push-minimatch-image push-frontendapi-image push-backendapi-image push-mmlogicapi-image
-push-mmf-example-images: push-mmf-go-grpc-serving-simple-image
-push-client-images: push-backendclient-image push-clientloadgen-image push-frontendclient-image
-push-evaluator-example-images: push-evaluator-serving-image
+push-images: push-service-images deprecated-push-images
+push-service-images: push-backend-image push-frontend-image  push-mmlogic-image
 
+push-backend-image: docker build-backend-image
+	docker push $(REGISTRY)/openmatch-backend:$(TAG)
+	docker push $(REGISTRY)/openmatch-backend:$(ALTERNATE_TAG)
+
+push-frontend-image: docker build-frontend-image
+	docker push $(REGISTRY)/openmatch-frontend:$(TAG)
+	docker push $(REGISTRY)/openmatch-frontend:$(ALTERNATE_TAG)
+
+push-mmlogic-image: docker build-mmlogic-image
+	docker push $(REGISTRY)/openmatch-mmlogic:$(TAG)
+	docker push $(REGISTRY)/openmatch-mmlogic:$(ALTERNATE_TAG)
+
+deprecated-push-images: deprecated-push-service-images deprecated-push-client-images deprecated-push-mmf-example-images deprecated-push-evaluator-example-images
+deprecated-push-service-images: push-minimatch-image push-frontendapi-image push-backendapi-image push-mmlogicapi-image
+deprecated-push-mmf-example-images: push-mmf-go-grpc-serving-simple-image
+deprecated-push-client-images: push-backendclient-image push-clientloadgen-image push-frontendclient-image
+deprecated-push-evaluator-example-images: push-evaluator-serving-image
+
+# Deprecated
 push-minimatch-image: docker build-minimatch-image
 	docker push $(REGISTRY)/openmatch-minimatch:$(TAG)
 	docker push $(REGISTRY)/openmatch-minimatch:$(ALTERNATE_TAG)
 
+# Deprecated
 push-frontendapi-image: docker build-frontendapi-image
 	docker push $(REGISTRY)/openmatch-frontendapi:$(TAG)
```

Comment:
Could we rename the deprecated image repo? It's hard to distinguish the deprecated ones from current naming. e.g. rename the deprecated image repo to deprecated-frontend or the new ones to future-frontend.